res/css/views/right_panel/_UserInfo.scss

```diff
@@ -104,18 +104,6 @@ limitations under the License.
         height: 100% !important;
     }
 
-    .mx_UserInfo_avatar .mx_BaseAvatar_initial {
-        z-index: 1;
-        display: flex;
-        align-items: center;
-        justify-content: center;
-
-        // override the calculated sizes so that the letter isn't HUGE
-        font-size: 6rem !important;
```

Comment:
don't we need to maintain such a max font-size so the font doesn't get much larger as part of this PR?

Reply:
![image](https://user-images.githubusercontent.com/2403652/120635975-fa4c2500-c464-11eb-9c3b-db698ab08ee3.png)

Yeah :D